magwest/templates/preregistration/disclaimers.html

```diff
@@ -2,13 +2,13 @@
 <h4 class="text-danger">Disclaimers</h4>
 <ul>
   <li>This is a one-time non-recurring payment. This is not a subscription service.</li>
-  <li> Please review our badge polices and other <a href="http://magwest.org/faq" target="_faq">frequently asked questions</a> prior to purchasing your badge for important information. </li>
+  <li> Please review our badge polices and other <a href="http://magwest.org/faq" target="_blank">frequently asked questions</a> prior to purchasing your badge for important information. </li>
   <li> If you don't pay for your registration, it will be deleted and you will have to pay full price at the door. </li>
-  <li> Your registration is <a href="http://magwest.org/faq" target="_faq">non-refundable</a> but may be <a href="http://magwest.org/faq#badge-transfer" target="_faq">sold/transferred</a>. </li>
-  <li> MAGFest has a <a href="http://magwest.org/faq" target="_faq#badge-for-someone-else">one badge per person policy</a>. If you purchase multiple badges (for your family, as a gift), please ensure that each badge is purchased in the name of the individual who the badge is intended for. You may use the same email address for multiple badges so any surprises aren't ruined. </li>
-  <li> Attendees (including dealers) must abide by our <a href="https://www.magwest.org/code-of-conduct" target="_coc">code of conduct</a>. </li>
+  <li> Your registration is <a href="http://magwest.org/faq/#badge-refunds" target="_blank">non-refundable</a> but may be <a href="http://magwest.org/faq#badge-transfer" target="_faq">sold/transferred</a>. </li>
+  <li> MAGFest has a <a href="https://www.magwest.org/faq/#multiple-badges" target="_blank">one badge per person policy</a>. If you purchase multiple badges (for your family, as a gift), please ensure that each badge is purchased in the name of the individual who the badge is intended for. You may use the same email address for multiple badges so any surprises aren't ruined. </li>
```

Comment:
You added https:// in this link, but in none of the others. Should they all be chagned?

Reply:
They should! We had only updated that line is why. I changed them all. Thanks for the catch!